apps/heft/src/plugins/SassTypingsPlugin/SassTypingsGenerator.ts

```diff
@@ -109,14 +114,14 @@ export class SassTypingsGenerator extends StringValuesTypingsGenerator {
     fileContents: string,
     filePath: string,
     buildFolder: string,
-    includePaths: string[] | undefined
+    importPaths: string[] | undefined
   ): Promise<string> {
     const result: Result = await LegacyAdapters.convertCallbackToPromise(render, {
       data: fileContents,
       file: filePath,
       importer: (url: string) => ({ file: this._patchSassUrl(url) }),
-      includePaths: includePaths
-        ? includePaths
+      includePaths: importPaths
```

Comment:
I agree with the change to label it "importPaths", but will this cause confusion for devs used to providing "includePaths" to `node-sass`? Just thinking out loud if the breaking change is the right call here.